CanHazFunny/CanHazFunny/Jester.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CanHazFunny
+{
+    public class Jester
+    {
+        private IJokeOutput outputService;
+        private IJokeService jokeService;
+        public IJokeOutput JokeOutputService { get => outputService; set => outputService = value; }
+        public IJokeService JokeService { get => jokeService; set => jokeService = value; }
```

Comment:
Given that these properties don't add any logic in the getter or setter a simply auto property would be cleaner.
```suggestion
        public IJokeOutput JokeOutputService { get; set; }
        public IJokeService JokeService { get; set; }
```